cookbook/typescript-agent/flow.ts

```diff
@@ -0,0 +1,14 @@
+import { Flow } from "../../typescript/brainyflow";
```

Comment:
```suggestion
import { Flow } from "brainyflow";
```

Reply:
we should make our code easy to copy/paste.
it's better to not use relative paths in import statements, and then use npm link to handle relative paths for us.

at the end, my local package.json contain a line similar to `"brainyflow": "link:../../typescript/brainyflow", which is a way to centralize the solution in a single line for the whole project.